src/current/cockroachcloud/troubleshooting-page.md

```diff
@@ -162,6 +162,14 @@ Using `sslmode=require` can leave your cluster vulnerable to MITM and impersonat
 
 ## CockroachDB {{ site.data.products.serverless }}
 
+### Delayed cluster access
+
+To enhance security, CockroachDB {{ site.data.products.serverless }} uses authentication throttling tracked per proxy and per (client IP, serverless cluster) pair. This means if multiple login attempts fail, access is temporarily delayed, with the wait time increasing after each attempt (up to an hour). Wrong passwords, brute force attacks, or SQL proxy rollouts may activate the throttling mechanism.
```

Comment:
SQL Proxy rollouts should not trigger the throttling mechanism, or it's a bug. Can we remove this?

Reply:
According to the issue description, SQL Proxy rollouts can trigger the mechanism: https://cockroachlabs.atlassian.net/browse/DOC-8686 
(I might have read that wrong tho)

Reply:
Oh, this might already be [fixed](https://cockroachlabs.atlassian.net/browse/CC-25314). Is the note still relevant for wrong passwords and brute force attacks?

Reply:
@daniel-crlabs Oh, this might already be [fixed](https://cockroachlabs.atlassian.net/browse/CC-25314). Is the note still relevant for wrong passwords and brute force attacks?

Reply:
This is an edge case that I don't think is worth documenting. If we do document it, we need to clarify the phrasing because rollouts are not the primary cause of the throttling. 

The sql proxy remembers successful authentications. So if a client IP address has ever successfully connected to a specific cluster, future connections from that client IP will not be throttled, even if the client IP starts to generate authentication errors.

The fact that a client ip has authenticated in the past is tracked in memory. So sometimes a user will start seeing a throttling error after a roll out, but only if their password is misconfigured. The primary cause is the misconfiguration, but the roll out is why they start seeing throttling errors instead of only seeing authentication failed errors.

Reply:
The throttle still exists, but it only kicks in if the user generates authentication failures. If the sql server is down, that is no longer counted as an authentication failure.

Reply:
I don't think it's worth making such a fine distinction in our docs. SQL proxy rollouts are an internal detail, and the impact here is only in the case where there's a misconfigured password, which the docs mention already.

Reply:
Removed the reference to SQL Proxy rollouts